frontend/src/router/index.ts

```diff
@@ -80,35 +85,36 @@ const routes: RouteRecordRaw[] = [
     ]
   },
   {
-    path: '/unlock-success',
+    path: '/app/unlock-success',
     component: UnlockSuccess,
     props: (route) => ({ vaultId: route.query.vault, deviceId: route.query.device })
   },
   {
-    path: '/unlock-error',
+    path: '/app/unlock-error',
     component: UnlockError,
     meta: { skipAuth: true }
   },
   {
-    path: '/:catchAll(.*)', //necessary due to using history mode in router
+    path: '/app/:pathMatch(.*)', //necessary due to using history mode in router
```

Comment:
```suggestion
    path: '/:pathMatch(.*)', //necessary due to using history mode in router
```

Should we switch it to `/` so that also something like `http://localhost:3000/foo` shows the 404 error page? In the current implementation it throws the following error 

```
[Vue Router warn]: No match found for location with path "/foo" vue-router.mjs:35:17
Uncaught (in promise) Error: No match for
 {"query":{"sync_me":null},"params":{}}
while being at
{"fullPath":"/foo","path":"/foo","query":{},"hash":"","params":{},"matched":[],"meta":{},"href":"/foo"}
    createRouterError vue-router.mjs:799
    resolve vue-router.mjs:1444
    resolve vue-router.mjs:3024
    <anonymous> index.ts:117
    promise callback* index.ts:115
    guardToPromiseFn vue-router.mjs:1979
    guardToPromiseFn vue-router.mjs:1953
    runGuardQueue vue-router.mjs:3536
    promise callback*runGuardQueue/< vue-router.mjs:3536
    runGuardQueue vue-router.mjs:3536
    navigate vue-router.mjs:3218
    promise callback*navigate vue-router.mjs:3211
    pushWithRedirect vue-router.mjs:3143
    push vue-router.mjs:3078
    install vue-router.mjs:3499
    use runtime-core.esm-bundler.js:4327
    <anonymous> main.ts:19
vue-router.mjs:799:22
```

Reply:
Tried, but behaves incorrectly as it collides with "real" 404 responses from the server. The Vue router should stay within its bounds.